src/pickley/bstrap.py

```diff
@@ -221,6 +221,9 @@ def seed_mirror(mirror, path, section):
             if not hdry(f"Would seed {msg}"):
                 print(f"Seeding {msg}")
                 with open(config_path, "wt") as fh:
+                    if section == "pip" and not mirror.startswith('"'):
+                        mirror = f'"{mirror}"'
```

Comment:
I don't want to dwell here, but the spec is very clear on what characters are allowed in strings:

https://toml.io/en/v1.0.0#string

This doesn't escape any of those, for example a quote in the middle.

Reply:
Noted, I'll see if I can provide something better here later, this is taking the optional value given as `--mirror` and "seeding" it to `uv.toml` and `pip.conf` for convenience, the script `bstrap.py` is also running with std lib only.